### ISSUE TITLE
[Bug]: Various issues running examples on windows

### ISSUE THREAD

**Issue by peakschris** [aspect-build/rules_lint#294]
### What happened?

There are various issues when running the example repo on Windows. It would be nice to fix these and give a better starting experience.

### Various linters/formatters don't work:
- clang-tidy/clang-format don't have windows target preconfigured
- buf fails
- etc

### Issues due to lack of windows support in other bazel modules
- bash lint.sh --fix //src:hello_cc fails on windows due to missing node.bat:
```
FATAL: aspect_rules_js[js_binary]: node wrapper '/d/udu/b/l2x2bhay/execroot/_main/./external/aspect_rules_lint~/lint/private/patcher_node_bin/node.bat' not found
```
- bazel run //docs:update_10 fails:
```
Target //docs:update_10 failed to build
Use --verbose_failures to see the command lines of failed build steps.
ERROR: Analysis of target '//docs:update_10' failed; build aborted: Target //docs:update_10 is incompatible and cannot be built, but was explicitly requested.
Dependency chain:
    //docs:update_10 (2bc110)
    //docs:clang-tidy-docgen.md (2bc110)
    //docs:clang-tidy (2bc110)   <-- target platform (@@local_config_platform//:host) didn't satisfy constraint @@platforms//:incompatible
```

### Issues local to examples repo
- example/.bazeliskrc causes builds to fail as aspect-cli doesn't support windows. A workaround is to delete this file.
- example/.bazelversion causes builds to fail as bazelisk doesn't support ../ notation. A workaround is to copy the parent .bazelversion into examples.
- lint.sh only returns a single target result, see https://github.com/aspect-build/rules_lint/issues/178

### Version

Development (host) and target OS/architectures:

Output of `bazel --version`:

7.1.1

Version of the Aspect rules, or other relevant rules from your
`WORKSPACE` or `MODULE.bazel` file:

rules_lint latest

Language(s) and/or frameworks involved:

all

### How to reproduce

```shell
Clone repo on windows
cd example
set BAZEL_SH=c:\msys64\usr\bin\bash.exe
get jq.exe
bazel test //...
```

**Comment by peakschris** [comment]
multitool PR: https://github.com/theoremlp/rules_multitool/pull/45
toolchains_llvm PR: https://github.com/bazel-contrib/toolchains_llvm/pull/342

**Comment by peakschris** [comment]
rules_multirun issue: https://github.com/keith/rules_multirun/issues/56

**Comment by peakschris** [comment]
I fixed a bunch of things in this and other repos, but some of the fixes didn't get anywhere. We are not currently using rules_lint for this reason. It would be good to have an up-to-date set of issues, could you supply this @albertocavalcante?

**Comment by thesayyn** [comment]
> example/.bazeliskrc causes builds to fail as aspect-cli doesn't support windows 

This is not true anymore, https://github.com/aspect-build/aspect-cli/releases/tag/2025.11.0 contains windows builds.

**Comment by thesayyn** [comment]
Our support for windows is really limited so i am not surprised that there are issues, natural next step for us is to turn on windows CI and try to keep it green.
